doc/memos/rdm0.md

```diff
@@ -0,0 +1,148 @@
+RIOT Developer Memo Format, Publishing and Maintenance Process
+=========
+
+- RIOT Developer Memo: 0  
+- Author(s): Emmanuel Baccelli  
+- Date: December 2016
+
+## Abstract
+
+This memo describes a template for memos produced by the RIOT maintainers 
+community, targeting primarily RIOT developers.
+
+
+## Status
+
+This document is a product of the community of RIOT maintainers, and aims to 
+represent the consensus within this community.
+The content of this document is licensed with a Creative Commons CC-BY-SA 
+license.
+
+## Terminology
+
+This memo uses the following terminology and acronyms:
+
+- RIOT-MC: RIOT Maintainers Community
+- RDM: RIOT Developer Memo
+- PR: git Pull-Request to RIOT's master branch on GitHub 
+- ACK: acknowledgement from a RIOT maintainer, concerning a pull-request to the 
+master branch of RIOT on GitHub
+
+# 1. Introduction
+
+In order to facilitate RIOT maintenance in the long term and at large scale, 
+additional documentation complementing code and usual code documentation is 
+needed. For instance, such memos are expected to describe and give explanations 
+about architectural design decisions, code structure etc. Other memos might
+also describe other aspects of RIOT activity, including, but not limited to,
+RIOT community processes, position with respect to some related external
+technical debates etc.
+
+
+This memo specifies a format for such memos and the process of discussing their 
+content before publication. Format and processes are somewhat inspired by 
+[IETF](http://ietf.org/) processes and documents such as Request For Comments 
+(RFC).
+
+# 2. RIOT Developers Memo Format
+
+RIOT Developer Memos (RDM) are written in English, using the Markdown (.md) 
+format, with less than 80 characters per line. Each memo has a unique 
+identification number indicated in its header, 
+e.g. this memo is RDM0.
```

Comment:
I know this has been debated a lot above. But my opinion is that additionally a brief title in the document name would also be helpful. Looking over a bunch of document numbers won't allow people to get a quick overview of what information resources are available. (Unless there's an index, which we then have to maintain.)
Something like: https://github.com/rust-lang/rfcs/tree/master/text

Reply:
I agree: I think a short title would be useful indeed!